### PR TITLE
[9.0] Port res_partner_attributes_add_SIN

### PR DESCRIPTION
**Pull request by JulieSFL**

### CODE REVIEW

File: res_partner_attributes_add_SIN/models/res_partner.py [hunk 1]
```diff
@@ -34,7 +17,7 @@ class ResPartner(models.Model):
         help="Social Insurance Number (9 digits)",
     )
 
-    @api.one
+    @api.multi
     @api.onchange("sin")
     @api.constrains("sin")
     def validate_sin(self):
```
Comment (joaoalf) [inline res_partner_attributes_add_SIN/models/res_partner.py]: @JulieSFL I'm not sure that the `@api.onchange` is really useful. If we use the method only as a constrain we would be able to use a more standard name for the method like `_check_sin(self)`. Also, I'm not very found on onchange methods that raises exceptions.

**Comment by joaoalf** [comment]
The problem with num2word is unrelated with this PR. For futher details check https://github.com/savoirfairelinux/num2words/issues/53

**Comment by Ehtaga** [comment]
Good :+1:

File: res_partner_attributes_add_SIN/__init__.py [hunk 2]
```diff
@@ -1,23 +1,5 @@
...
-#    but WITHOUT ANY WARRANTY; without even the implied warranty of
-#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-#    GNU Affero General Public License for more details.
-#
-#    You should have received a copy of the GNU Affero General Public License
-#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-##############################################################################
+# -*- coding: utf-8 -*-
+# © 2016 Savoir-faire Linux
```
Comment (jbchouinard) [inline res_partner_attributes_add_SIN/__init__.py:2]: `# © 2010-2014, 2016 Savoir-faire Linux`

File: res_partner_attributes_add_SIN/__openerp__.py [hunk 3]
```diff
@@ -1,28 +1,10 @@
...
-#    This program is distributed in the hope that it will be useful,
-#    but WITHOUT ANY WARRANTY; without even the implied warranty of
-#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-#    GNU Affero General Public License for more details.
-#
-#    You should have received a copy of the GNU Affero General Public License
-#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-##############################################################################
+# © 2016 Savoir-faire Linux
```
Comment (jbchouinard) [inline res_partner_attributes_add_SIN/__openerp__.py:2]: `# © 2010-2014, 2016 Savoir-faire Linux`

**Review by Ehtaga — APPROVED** [review]

File: res_partner_attributes_add_SIN/__openerp__.py (hunk 3 above)
Comment (ghost) [reply res_partner_attributes_add_SIN/__openerp__.py:2]: je suis pas contre d'utiliser un range de dates, mais 2010-2014, je suis presque certain que c'est pas bon. Le premier commit du projet remonte à 2012.

`
commit 058ff50a307d35645df6707fe17ed94102a88062
Author: Maxime Chambreuil maxime.chambreuil@savoirfairelinux.com
Date:   Fri Mar 9 18:23:14 2012 -0500

```
[ADD] Initial commit. Import from extra-addons.
```

`

Comment (JulieSFL) [reply res_partner_attributes_add_SIN/__openerp__.py:2]: @dufresnedavid  je mets  # © 2012-2014, 2016 Savoir-faire Linux  ?
Les autres ont été mergés avec # © 2016 Savoir-faire Linux

Comment (ghost) [reply res_partner_attributes_add_SIN/__openerp__.py:2]: @JulieSFL 2012-2016 stp

Comment (joaoalf) [reply res_partner_attributes_add_SIN/__openerp__.py:2]: @JulieSFL @dufresnedavid @jbchouinard AFAIK

(C) 2014 - The work was published the first time in 2014 (The copyright claim is from 2014 onward)
(C) 2014-2016 - The work was first published in 2014 and last updated in 2016 (The copyright claim is from 2014 onward)
(C) 2014,2015,2016 - The work was first published in 2014 and updated in 2015 and 2016 (The copyright claim is from 2014 onward)
(C) 2010-2014,2016 Not sure but the copyright claim is from 2010 onward.

The most simple way is to just put the year of the first publication.

Comment (joaoalf) [reply res_partner_attributes_add_SIN/__openerp__.py:2]: @Ehtaga @dufresnedavid I'm OK with the format (C) 2016 Savoir-faire Linux. Can we merge this?

**Comment by ghost** [comment]
:+1:

**Pull request merged**